include/fizzy/fizzy.h

```diff
@@ -50,11 +50,31 @@ typedef struct FizzyExecutionResult
 typedef FizzyExecutionResult (*FizzyExternalFn)(void* context, FizzyInstance* instance,
     const union FizzyValue* args, size_t args_size, int depth);
 
+/// Value type.
+typedef uint8_t FizzyValueType;
+static const FizzyValueType FizzyValueTypeI32 = 0x7f;
+static const FizzyValueType FizzyValueTypeI64 = 0x7e;
+static const FizzyValueType FizzyValueTypeF32 = 0x7d;
+static const FizzyValueType FizzyValueTypeF64 = 0x7c;
+
+/// Function type.
+typedef struct FizzyFunctionType
+{
+    /// Pointer to input types array.
+    const FizzyValueType* inputs;
+    /// Input types array size.
+    size_t inputs_size;
+    /// Pointer to output types array.
+    const FizzyValueType* outputs;
```

Comment:
This is ok, but also can be greatly simplified by defining `FizzyValueTypeVoid` and using `FizzyValueType output_type`.

Reply:
Good idea, I'll try it.

Reply:
Move it to the front.